.platform.app.yaml

```diff
@@ -16,7 +16,7 @@ type: 'php:7.2'
 # side is in the form `<service name>:<endpoint name>`.
 relationships:
     database: 'db:mysql'
-    redis: 'cache:redis'
+    redis: 'cache:redis'#
```

Comment:
Is this `#` right?

Reply:
Not in the least. :smile:   Rebuilding now.